src/com/kandbnutrition/controller/CustomerFitTrackerCell.java

```diff
@@ -0,0 +1,49 @@
+package com.kandbnutrition.controller;
+
+/*
+ * Created by Kyle Wolff 1/21/2016
+ * 
+ * This is the controller for the custom cell used in the FitTrackerView
+ */
+
+import java.io.IOException;
+
+import com.kandbnutrition.model.FitTrackerData;
+import com.kandbnutrition.resource.StringValues;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.TextField;
+import javafx.scene.shape.Line;
+
+public class CustomerFitTrackerCell extends ListCell<FitTrackerData> {
+	
+	@FXML 
+	Line leftLine, rightLine;
+	
+	@FXML
+	TextField exerciseTextField,  weightTextField, repsTextField, setsTextField;
+	
+	private StringValues stringValues;
+	
+	public CustomerFitTrackerCell() {
+		
+		stringValues = new StringValues();
```

Comment:
We could make another enum? Static is what I am trying not to do.
On Jan 21, 2016 1:55 PM, "Brandon VanderMey" <notifications@github.com>
wrote:

> In src/com/kandbnutrition/controller/CustomerFitTrackerCell.java
> <https://github.com/TheLearningCurve/NutritionApiV3/pull/27#discussion_r50444995>
> :
>
> > +import javafx.scene.control.TextField;
> > +import javafx.scene.shape.Line;
> > +
> > +public class CustomerFitTrackerCell extends ListCell<FitTrackerData> {
> > +	
> > +	@FXML
> > +	Line leftLine, rightLine;
> > +	
> > +	@FXML
> > +	TextField exerciseTextField,  weightTextField, repsTextField, setsTextField;
> > +	
> > +	private StringValues stringValues;
> > +	
> > +	public CustomerFitTrackerCell() {
> > +		
> > +		stringValues = new StringValues();
>
> I don't like how we have to create a new object to retrieve string values.
> We should access this statically.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TheLearningCurve/NutritionApiV3/pull/27/files#r50444995>
> .
>